updating

csoleimani opened the pull request:
# Pull Request Checklist

<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Related Issue
<!--- Suggest creating an issue first and then referencing it here. -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

Fixes # .

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## General

- [ ] Update `README.md` with any necessary configuration snippets

- [ ] New parameters are documented

- [ ] New parameters have tests

- [ ] Tests pass - `bundle exec rake validate lint spec`

alvagante commented:
there are also some additions commits from a more recent master , but I suppose you'll have to squash everything in any case

alvagante commented:
After a better look at what is done during provisioning on Vagrant, I think you just needed to have ensure = present when $::sensu::transport_type == 'rabbitmq'.
No need of special selector for MacOS

alvagante commented on manifests/transport.pp:
```diff
@@ -5,13 +5,14 @@
+  or $::sensu::transport_type == 'rabbitmq' {
+    $ensure = 'present'
+  } else {
     $ensure = $::osfamily ? {
```
This selector is probably not needed anymore if you add the or $::sensu::transport_type == 'rabbitmq' line.

csoleimani commented:
Sorry I've been on vacation. Just merged your changes - looking at the Vagrantfile thing now. Did you get the USB error on trying vagrant up sensu-server?

treydock committed:
Better support for resources in different namespaces

Resources that can belong to a namespace now support composite names in form of "<name> in <namespace>".

ghoneycutt committed:
Merge pull request #1126 from sensu/namespaces

Better support for resources in different namespaces

ghoneycutt committed:
Release v3.6.0
